### ISSUE TITLE
MONGO_PORT config via environment variables

### ISSUE THREAD

**Issue by TooAngel** [ScreepsMods/screepsmod-mongo#1]
Server version: 2.7.2
screepsmod-mongo version: 1.0.0

Changing the MONGO_PORT via environment variables is currently not possible. As far as I can see the screeps server process cleans up the environment variables.

`console.log(process.env)` with the `lib/common/index.js`

```
{ GAME_PORT: '21025',
  GAME_HOST: '0.0.0.0',
  CLI_PORT: '21026',
  CLI_HOST: 'localhost',
  STORAGE_PORT: '21027',
  MODFILE: 'mods.json',
  ASSET_DIR: 'assets',
  SERVER_PASSWORD: '',
  STEAM_KEY: 'SUPERSECRET'}
```

Wondering why not having the default mongo db port `27017` as default port in your config.

**Comment by ags131** [comment]
Fixed in 3e737a4, added README note since env vars don't apply when using the default launcher

**Comment by ags131** [comment]
Port was also updated to 27017, 28017 was a typo on my part, but happened to work for some reason
